src/query/api/v1/handler/prometheus/native/read.go

```diff
@@ -128,8 +128,7 @@ func (h *PromReadHandler) read(
 
 	opts := &executor.EngineOptions{}
 	// Detect clients closing connections
-	abortCh, _ := handler.CloseWatcher(ctx, w)
-	opts.AbortCh = abortCh
```

Comment:
Is `abortCh` now unused? Could we eliminate it entirely (in favor of using contexts like this PR does)?

Reply:
will remove